src/platform/plugins/shared/management/public/components/landing/landing.tsx

```diff
@@ -25,10 +27,24 @@ export const ManagementLandingPage = ({
   setBreadcrumbs,
   onAppMounted,
 }: ManagementLandingPageProps) => {
-  const { appBasePath, sections, kibanaVersion, cardsNavigationConfig, chromeStyle, coreStart } =
-    useAppContext();
+  const { euiTheme } = useEuiTheme();
+  const {
+    appBasePath,
+    sections,
+    kibanaVersion,
+    cardsNavigationConfig,
+    chromeStyle,
+    coreStart,
+    cloud,
+    hasEnterpriseLicense,
+  } = useAppContext();
   setBreadcrumbs();
 
+  // Check if cloud services are available
+  const isCloudEnabled = cloud?.isCloudEnabled || false;
+  // AutoOps promotion callout should only be shown for self-managed instances with an enterprise license
+  const shouldShowAutoOpsPromotion = !isCloudEnabled && hasEnterpriseLicense;
```

Comment:
I do wonder if we should maintain state for removing this callout, at least in the browser. It may prove to be very annoying to constantly remove this, especially if they actually do make use of AutoOps.

Reply:
Ok, I've added some small changes to persist the closed state with a localstorage variable, later on with more time we can make it into a SO variable so it persist as a user setting